rust/src/ffi/evmc_vm.rs

```diff
@@ -0,0 +1,211 @@
+use std::{
+    ffi::{c_char, CStr},
+    panic, process, slice,
+};
+
+use evmc_vm::{
+    ffi::{
+        evmc_capabilities, evmc_capabilities_flagset, evmc_host_context, evmc_host_interface,
+        evmc_message, evmc_result, evmc_revision, evmc_set_option_result, evmc_status_code,
+        evmc_vm as evmc_vm_t, EVMC_ABI_VERSION,
+    },
+    EvmcContainer, EvmcVm, ExecutionContext, ExecutionMessage, ExecutionResult, SetOptionError,
+};
+
+use crate::evmc::EvmRs;
+
+static EVM_RS_NAME: &str = "evmrs\0";
+static EVM_RS_VERSION: &str = "0.1.0\0";
+
+pub const EVMC_CAPABILITY: evmc_capabilities = evmc_capabilities::EVMC_CAPABILITY_EVM1;
```

Comment:
A comment explaining why is EVM1 would be nice to have.

Reply:
Sure